src/bridge/layouts/mod.rs

```diff
@@ -64,20 +64,20 @@ fn append_modifiers(
     if result == "<" {
         result = "lt".to_string();
         special = true;
-    }
+    }    
 
     if shift {
         special = true;
         result = format!("S-{}", result);
     }
-    if ctrl {
+	if ctrl && !alt {
         special = true;
         result = format!("C-{}", result);
     }
-    if alt {
+    if alt && !ctrl {
         special = true;
         result = format!("M-{}", result);
-    }
+	}
```

Comment:
This breaks C-M-{} bindings on keyboard layouts which allow such bindings

Reply:
Hmm I see, thats interesting. I'll investigate further, as I noted I more or less stole this change from neovim-qt but perhaps they don't handle C-M-{} bindings either? In any case I'll have to further investigate why `[ ] @ \ ` and a few other keys weren't working for me in the first place.

Reply:
Interesting. Its very possible that I don't understand keyboard input in neovim yet either :(